src/components/candidateContent/candidateFeedbacksItem/CandidateFeedbacksItem.jsx

```diff
@@ -18,25 +18,34 @@ import { updateFeedback, createFeedback } from '../../../store/commands';
 
 const CandidateFeedbacksItem = ({ user, candidateInfo }) => {
   const dispatch = useDispatch();
-  const [isCriteriaShown, setIsCriteriaShown] = React.useState(false);
+  const [isCriteriaShown, setIsCriteriaShown] = React.useState(true);
   const now = new Date(Date.now());
   const { feedbacks, userName, roleType } = user;
   const feedback = feedbacks.length ? feedbacks[0] : {};
+  const [description, setDescription] = React.useState(
+    feedback.description === '.' ? '' : feedback.description,
+  );
   const [finalEvaluation, setFinalEvaluation] = React.useState(
     feedbacks.length ? feedback.finalEvaluation : 0,
   );
   const [editMode, setEditMode] = React.useState(false);
 
-  const updateToNewFeedback = () => ({
-    id: feedback.id,
-    userId: feedback.userId,
-    candidateId: feedback.candidateId,
-    englishLevelType: feedback.englishLevelType,
-    date: now.toISOString(),
-    description: '.',
-    evaluations: [],
-    finalEvaluation,
-  });
+  const updateToNewFeedback = () => {
+    const newFeedback = {
+      id: feedback.id,
+      userId: feedback.userId,
+      candidateId: feedback.candidateId,
+      englishLevelType: feedback.englishLevelType,
+      date: now.toISOString(),
+      description,
+      evaluations: [],
+      finalEvaluation,
+    };
+    if (!newFeedback.description) {
+      newFeedback.description = '.';
```

Comment:
???? How it works

Reply:
This is a shortcut to the backend. They accept the descriptions field as required, but it can be empty.